backend/build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+
+
+
+
+plugins {
+    // TODO: Move ktLint and jvm to Convention
```

Comment:
You have a comment with a pending task, if you have already resolved it, just delete the comment